### PR TITLE
EMIP Metadata

### PR DESCRIPTION
**Pull request by Ralph-Lee-UK**
New work files as discussed in https://github.com/BetaMasaheft/Documentation/issues/1515

I have attached a copy of the PDF in case you need to check anything. 

[EMIP00764_ChalakotSilasie23-compressed.pdf](https://github.com/BetaMasaheft/Works/files/6089471/EMIP00764_ChalakotSilasie23-compressed.pdf)

### CODE REVIEW

File: new/LIT6389MalkeaTewodros.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6389MalkeaTewodros" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መልክአ፡ ቴዎድሮስ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Mälkǝʿa Tewodros</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6389MalkeaTewodros.xml:9]: Malkǝʿa

File: new/LIT6390HymnVictor.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6390HymnVictor" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ወይነ፡ መንፈስ፡ ቅዱስ፡ ፊቅጦር፡ ዘጣዕምከ፡ ፍድፉድ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Wäynä mänfäs qǝddus Fiqṭor zä-ṭāʿǝmkä fǝdfud</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6390HymnVictor.xml:9]: Please note we use the opposition a vs ā.

File: new/LIT6390HymnVictor.xml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+          <div type="bibliography">
+            <div type="edition">
+              <note>the text follows <ref type="mss" corresp="EMIP00764"/> ff. 116r-118v</note>
+              <div type="textpart" subtype="incipit" xml:lang="gez">
+                <ab>
+                  በረድኤተ፡ ክርስቶስ፡ አምላክ፡ ለክብረ፡ ስሙ፡ ስጊድ።
```
Comment (DenisNosnitsin1970) [inline new/LIT6390HymnVictor.xml:60]: I think it is ሰጊድ

File: new/LIT6391HymnGabraMasqal.xml [hunk 4]
```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6391HymnGabraMasqal" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ገብረ፡ መስቀል፡ ቅላዕ፡ ባሕቱ፡ መንጦላዕተ፡ ግዙፍ፡ ልብየ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Gäbrä Mäsqäl qǝlāʿe bāḥǝttu mänṭolāʿǝtä gǝzuf lǝbbǝyä</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6391HymnGabraMasqal.xml:9]: We use the opposition a vs ā.

File: new/LIT6391HymnGabraMasqal.xml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                  ወበስመ፡ ማርያም፡ ድንግል፡ ዘተመሰለት፡ ሰማየ።
+                  ስብሐት፡ መልክዕከ፡ እንብብ፡ እንዘ፡ እከሥት፡ አፉየ።
+                  ገብረ፡ መስቀል፡ ቅላዕ፡ ባሕቱ፡ መንጦላዕተ፡ ግዙግ፡ ልብየ።
+                </ab>
+              </div>
+              <div type="textpart" subtype="explicit" xml:lang="gez">
+                <ab>
+                  ሰላም፡ ለከ፡ ኦአቡየ፡ መንፈሳዊ፡ ገብረ፡ መስቀል፡ ጻድቅ።
+                  አመ፡ ኵነኔ፡ ወድልቅልቅ።
+                  መግሥተ፡ ሰማያት፡ ምርሐኒ፡ ዘኢየኃልቅ።
```
Comment (DenisNosnitsin1970) [inline new/LIT6391HymnGabraMasqal.xml:70]: መንግሥተ

File: new/LIT6392HymnInnocents.xml [hunk 6]
```diff
@@ -0,0 +1,79 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6392HymnInnocents" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሕፃናት፡ ኄራን፡ ወንጹሐን፡ እምርኵስ፡ ናሁ፡ ወጠንኩ፡ ሰላመክሙ፡ በብርሃነ፡ ጥበብ፡ ቅዱስ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Ḥǝḍānāt ḫerān wä-neṣuḥan ʾǝmerǝkwus nāhu wäṭänku sälāmäkemu bä-bǝrhanä ṭǝbäb qǝddus</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6392HymnInnocents.xml:9]: Please correct the transliteration. The opposition long and short "a"; ʾǝmerǝkwus > ʾǝm-rǝkwus, as far as I understand, and labialized k with raise w (see https://betamasaheft.eu/Guidelines/?id=transliteration-principles).

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT6389MalkeaTewodros.xml (hunk 1 above)
Comment (Ralph-Lee-UK) [reply new/LIT6389MalkeaTewodros.xml:9]: Thanks for these! I was transliterating Amharic a la Appleyard, and it muddled me!

File: new/LIT6390HymnVictor.xml (hunk 3 above)
Comment (Ralph-Lee-UK) [reply new/LIT6390HymnVictor.xml]: Correct!

File: new/LIT6390HymnVictor.xml [hunk 7]
```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6390HymnVictor" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ወይነ፡ መንፈስ፡ ቅዱስ፡ ፊቅጦር፡ ዘጣዕምከ፡ ፍድፉድ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Wayna mänfas qǝddus Fiqṭor za-ṭāʿǝmka fǝdfud</title>
+                <title xml:lang="en" corresp="#t1">Victor, vine of the Holy Spirit whose flavour is excellent</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6390HymnVictor.xml:10]: you have still mänfas here :)

File: new/LIT6391HymnGabraMasqal.xml [hunk 8]
```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6391HymnGabraMasqal" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ገብረ፡ መስቀል፡ ቅላዕ፡ ባሕቱ፡ መንጦላዕተ፡ ግዙፍ፡ ልብየ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Gabra Masqal qǝlāʿe bāḥǝttu manṭolāʿǝta gǝzuf lǝbbǝya</title>
+                <title xml:lang="en" corresp="#t1">Gäbrä Mäsqäl, just unveil the veil of my subborn heart</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6391HymnGabraMasqal.xml:10]: Please correct Gäbrä Mäsqäl

File: new/LIT6392HymnInnocents.xml [hunk 9]
```diff
@@ -0,0 +1,79 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6392HymnInnocents" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሕፃናት፡ ኄራን፡ ወንጹሐን፡ እምርኵስ፡ ናሁ፡ ወጠንኩ፡ ሰላመክሙ፡ በብርሃነ፡ ጥበብ፡ ቅዱስ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Ḥǝḍānāt ḫerān wa-neṣuḥan ʾǝm-erǝkʷus nāhu waṭanku salāmakemu ba-bǝrhana ṭǝbab qǝddus</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6392HymnInnocents.xml:9]: I think it is ʾǝm-rǝkʷus, not ʾǝm-erǝkʷus; check also salāmakemu (salāmakkǝmu?)

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT6390HymnVictor.xml (hunk 7 above)
Comment (Ralph-Lee-UK) [reply new/LIT6390HymnVictor.xml:10]: Sorry, Denis! Too much in a rush!

File: new/LIT6391HymnGabraMasqal.xml (hunk 8 above)
Comment (DenisNosnitsin1970) [reply new/LIT6391HymnGabraMasqal.xml]: manṭolāʿta

File: new/LIT6389MalkeaTewodros.xml [hunk 10]
```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6389MalkeaTewodros" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መልክአ፡ ቴዎድሮስ፡</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6389MalkeaTewodros.xml:8]: Glottal stop in transliteration?

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]
The last couple of remarks.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]
The last couple of remarks.

File: new/LIT6389MalkeaTewodros.xml (hunk 10 above)
Comment (Ralph-Lee-UK) [reply new/LIT6389MalkeaTewodros.xml:8]: @DenisNosnitsin1970 I took this transliteration from the way it is done in most of the Chaine Repertoire, so: Malkǝʾa śǝʿǝl, Malkǝʾa ʾEdom or have I missed something else? Sorry to drag this one out so much!

Comment (DenisNosnitsin1970) [reply new/LIT6389MalkeaTewodros.xml:8]: That is fine, I meant, you have transliteration "Malkǝʿa" with Ayn, Ethiopic with Alif.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT6389MalkeaTewodros.xml (hunk 10 above)
Comment (Ralph-Lee-UK) [reply new/LIT6389MalkeaTewodros.xml:8]: Aha! I should have spotted that!

Comment (Ralph-Lee-UK) [reply new/LIT6389MalkeaTewodros.xml:8]: Aha! I should have spotted that!

**Comment by Ralph-Lee-UK** [comment]
Phew!

**Pull request merged**